include/model_rsa.h

```diff
@@ -2,14 +2,13 @@
 
 #include "verilog/dtype/vint.h"
 #include "verilog/dtype/vstruct.h"
-#include <iostream>
-#include <systemc>
 
 constexpr int kBW = 256;
 using KeyType = verilog::vuint<kBW>;
+using ExtendKeyType = verilog::vuint<kBW + 1>;
```

Comment:
The `ExtendKeyType` is `kBW + 1` in TwoPower, and `kBW + 2` in Montgomery. 
So it might be a little unsuitable to put it here.